int64 frame_id
int32 frame_size
float32[150] P
float32[150] range
float32[150] vel
float32[150] ang
float32[150] ang_elv